### DISCUSSION TITLE
Support of Temporal

### DISCUSSION THREAD

**Discussion by jsejcksn** [denoland/deno#15321] [Q&A] [answered]
Are there plans/prior discussion to be among the early implementers of [tc39/proposal-temporal](https://github.com/tc39/proposal-temporal)?

**Comment by kitsonk** [comment, marked as answer]
Because it would be part of V8, we would implement it when it is in V8 stable, as we have other ECMA-262 APIs when available in V8.

This is the tracking bug: https://bugs.chromium.org/p/v8/issues/detail?id=11544
And this is the platform status link: https://chromestatus.com/feature/5668291307634688
